great_expectations/self_check/util.py

```diff
@@ -2004,3 +1970,9 @@ def generate_test_table_name(
         [random.choice(string.ascii_letters + string.digits) for _ in range(8)]
     )
     return table_name
+
+
+def _create_bigquery_engine() -> Engine:
+    gcp_project = os.getenv("GE_TEST_BIGQUERY_PROJECT", "superconductive-internal")
+    bigquery_dataset = os.getenv("GE_TEST_BIGQUERY_DATASET", "test_ci")
```

Comment:
Are these defaults the same as what's actually stored in the env vars? If so, maybe we remove the defaults and make a note that setting env vars is a prerequisite to getting things up and running here?

Reply:
Thank you for catching this.. I've cleaned this up, and added a helpful error message